kafka-client/src/main/java/dev/responsive/kafka/api/config/ResponsiveConfig.java

```diff
@@ -157,6 +157,31 @@ public class ResponsiveConfig extends AbstractConfig {
   private static final String SUBPARTITION_HASHER_DOC = "Hasher to use for sub-partitioning.";
   private static final Class<?> SUBPARTITION_HASHER_DEFAULT = Murmur3Hasher.class;
 
+
+  // ------------------ WindowStore configurations ----------------------
+
+  public static final String WINDOW_BLOOM_FILTER_ENABLED_CONFIG = "responsive.window.bloom.filter.enabled";
+  private static final boolean WINDOW_BLOOM_FILTER_ENABLED_DEFAULT = false;
+  private static final String WINDOW_BLOOM_FILTER_ENABLED_DOC = "Whether to enable a bloom filter for "
+      + "the most recent active window of a window aggregate. Only applies to hopping and tumbling "
+      + "windowed aggregations. If enabled, we highly recommend configuring the expected number of "
+      + "keys per partition with responsive.window.bloom.filter.expected.keys";
+
+  public static final String WINDOW_BLOOM_FILTER_EXPECTED_KEYS_CONFIG = "responsive.window.bloom.filter.expected.keys";
+  private static final long WINDOW_BLOOM_FILTER_EXPECTED_KEYS_DEFAULT = 1_000L;
+  private static final String WINDOW_BLOOM_FILTER_EXPECTED_KEYS_DOC = "Expected number of elements for "
+      + "the WindowStore bloom filters, or the approximate number of unique keys per partition that "
+      + "will be present in the most recent window of a hopping or tumbling windowed aggregation. "
+      + "For loads that vary over time, use the maximum keys per partition that may appear in a window. "
+      + "An overestimate will use slightly more memory, but an underestimate will significantly degrade "
+      + "the bloom filter's performance";
+
+  public static final String WINDOW_BLOOM_FILTER_FPP_CONFIG = "responsive.window.bloom.filter.fpp";
+  private static final double WINDOW_BLOOM_FILTER_FPP_DEFAULT = 0.03d;
```

Comment:
we can probably use a much lower fpp by default here (like 1% or .1%)

Reply:
nvm this is supposed to be a number between 0 and 1, so this is 3% fppp

Reply:
I guess? Would be nice to do some tests to find out how much memory is used. I just went with the actual default this class uses.
Anyways just FYI I'm going to split the bloom filter out into a separate PR so we can punt on this for a minute